test/e2e/node/mount_propagation.go

```diff
@@ -102,8 +105,8 @@ var _ = SIGDescribe("Mount propagation", func() {
 		// running in parallel.
 		hostDir := "/var/lib/kubelet/" + f.Namespace.Name
 		defer func() {
-			cleanCmd := fmt.Sprintf("sudo rm -rf %q", hostDir)
-			e2essh.IssueSSHCommand(cleanCmd, framework.TestContext.Provider, node)
```

Comment:
Should we remove `e2essh.IssueSSHCommand`, or just have it call hostexec under the hood?

Reply:
I'd assume just like real SSH we will need to handle re-connect?

Reply:
Yeah, there are solutions, but they are far more risky than good old ssh. I'd prefer gradual move to `hostexec`.